tests/test_money.py

```diff
@@ -153,3 +153,39 @@ def test_sub_negative():
     for bill in bills:
         total -= bill
     assert total == Money(-33, "EUR")
+
+
+def test_add_result_in_proper_instance():
+    one = Money(1, "EUR")
+    assert one._decimal_places_display is None, "this should be None"
+    assert one.decimal_places == 2
```

Comment:
Generally, I'd prefer a bit higher level assertions (at least in new tests) - e.g. how values will be formatted, as these attributes are telling more about implementation details.